build/dependencies.props

```diff
@@ -49,6 +49,8 @@
     <Tooling_NewtonsoftJsonPackageVersion>9.0.1</Tooling_NewtonsoftJsonPackageVersion>
     <XunitPackageVersion>2.3.1</XunitPackageVersion>
     <XunitRunnerVisualStudioPackageVersion>2.4.0-beta.1.build3945</XunitRunnerVisualStudioPackageVersion>
+
+    <StableMicrosoftAspNetCoreServerIISIntegrationPackageVersion>2.1.0-rc1-final</StableMicrosoftAspNetCoreServerIISIntegrationPackageVersion>
```

Comment:
/cc @natemcmaster, this is the only reference we need, I'll add it as 3rd party to universe.

Reply:
Can we call it `MicrosoftAspNetCoreServerIISIntegration21PackageVersion`?

Reply:
I dropped retargeting to 2.1 from this PR.